README.md

```diff
@@ -6,7 +6,18 @@
 >
 > -- anonymous
 
-* [Building npm packages in Nix](#building-npm-packages-in-nix)
+**Table of contents**
```

Comment:
I believe GitHub supports this natively now, though I were not able to found it in the settings.

Reply:
It is available on the left of the `README.md` label, although not many users use it. I think that adding it manually into README, makes it more accessible. I can remove it if you think that it is redundant.

![image](https://user-images.githubusercontent.com/22836301/130642295-6cf8eee3-9174-4e05-b0b7-4736ff1f17ef.png)